server/dev/user.clj

```diff
@@ -17,9 +17,30 @@
             [clojure.repl :refer :all]
             [com.stuartsierra.component :as component]
             [io.pedestal.http :as server]
-            [spacon.server :refer [make-spacon-server]]
+            [spacon.server :refer [make-spacon-server-mqtt make-spacon-server-kafka]]
             [clojure.tools.logging :as log]))
 
+(defn start-mqtt-system []
+  (log/info "starting mqtt system")
+  (make-spacon-server-mqtt {:http-config {:env                     :dev
+                                          ::server/join?           false
+                                          ::server/allowed-origins {:creds true
+                                                                    :allowed-origins (constantly true)}}
+                            :mqtt-config {:broker-url (or (System/getenv "MQTT_BROKER_URL")
+                                                          "tcp://localhost:1883")}})
+  )
```

Comment:
Probably want to move this paren to conform to the conventional style.  Try using [parinfer](https://shaunlebron.github.io/parinfer/index.html) and see how you like it.